### PR TITLE
roachprod: add ability to run preemptible insances in GCE

### PR DESCRIPTION
**Pull request by rail**
Previously, roachprod instances on GCE were created using default non
preemptible instances.

This patch adds ability to use `--gce-preemptible` in order to use
preemptible instances.

Release note: None

### CODE REVIEW

**Review by ajstorm — COMMENTED** [review]
:lgtm: Thanks for getting to this so quickly!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/68657#-:-MglBAL9A04B9dLi1cBG:b-s74pn1)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @jlinder)

File: pkg/cmd/roachprod/vm/gce/gcloud.go [hunk 1]
```diff
@@ -289,6 +291,7 @@ func (o *providerOpts) ConfigureCreateFlags(flags *pflag.FlagSet) {
 			"will be repeated N times. If > 1 zone specified, nodes will be geo-distributed\n"+
 			"regardless of geo (default [%s])",
 			strings.Join(defaultZones, ",")))
+	flags.BoolVar(&o.preemptible, ProviderName+"-preemptible", false, "use preemptible instances")
```
Comment (jlinder) [inline pkg/cmd/roachprod/vm/gce/gcloud.go:294]: Can we make it clear this is just for GCP clusters?

Comment (rail) [reply pkg/cmd/roachprod/vm/gce/gcloud.go:294]: In the help message? The actual flag printed by `--help` is `--gce-preemptible`.

Comment (rail) [reply pkg/cmd/roachprod/vm/gce/gcloud.go]: Added.

**Comment by rail** [comment]
bors r=ajstorm

**Pull request merged**